channel.go

```diff
@@ -134,6 +139,16 @@ type ChannelOptions struct {
 	ConnContext func(ctx context.Context, conn net.Conn) context.Context
 }
 
+// HandlerWithIgnore allows users to configure TChannel server such that
+// requests with method listed in NativeHanderMethods are
+// ignored by passed-in handler and natively by TChannel.
+// All other methods will be handled by passed-in handler.
+type HandlerWithIgnore struct {
+	Handler Handler
+	// should we generalize to a func(request) bool here?
```

Comment:
good question, since we have a struct here, we can always add more in the future, OK with starting simple